src/main/cpp/subsystems/StateSubsystem.cpp

```diff
@@ -157,9 +169,10 @@ frc2::CommandPtr StateSubsystem::StartClimb() {
       .AndThen(
           // TODO: method to get the stage location
```

Comment:
Stale comment. However, are we handling field flips yet when getting locations for on-the-fly PP?

Reply:
We are not, it's on the list of things that need to be handled before week 0

Reply:
Handled in one of my recent pushes